becquerel/core/peakfinder.py

```diff
@@ -389,15 +390,9 @@ def find_peaks(self, xmin=None, xmax=None, min_snr=2, max_num=40):
         max_num = int(max_num)
         if max_num < 1:
             raise PeakFinderError("Must keep at least 1 peak, not {}".format(max_num))
-        # calculate the first derivative and second derivatives of the SNR
-        d1 = (self.snr[2:] - self.snr[:-2]) / 2
-        d1 = np.append(0, d1)
-        d1 = np.append(d1, 0)
-        d2 = self.snr[2:] - 2 * self.snr[1:-1] + self.snr[:-2]
-        d2 = np.append(0, d2)
-        d2 = np.append(d2, 0)
+
         # find maxima
-        peak = (d1[2:] < 0) & (d1[:-2] > 0) & (d2[1:-1] < 0)
+        peak = (self.snr[2:] <= self.snr[1:-1]) & (self.snr[:-2] < self.snr[1:-1])
```

Comment:
I was initially hesitant to remove the first and second derivative logic here since it incorporates information from another index away and thus might be smoother and less noisy, however since `snr` is already smoothed it's a moot point.

Reply:
Optional, but what do you think about this ordering, I find it more logical:

`peak = (self.snr[:-2] < self.snr[1:-1]) & (self.snr[1:-1] >= self.snr[2:])`

Reply:
Sure